spec/spec_helper.rb

```diff
@@ -2,15 +2,18 @@
 
 ENV["RAILS_ENV"] = "test"
 
+require "rails"
```

Comment:
do we need rails?

Reply:
- Yes we need, `railties` was not enough.